src/main/scala/util/ExecuteOpenCL.scala

```diff
@@ -8,9 +8,10 @@ object ExecuteOpenCL {
   case class Exception(msg: String) extends Throwable
 
   val runtimePath = "runtime/"
+  val platformPath = "runtime/ocl/"
   val executorHeadersPath = "lib/executor/lib/Executor/include/"
   val libs = "-lm -lOpenCL"
-  val includes = s"-I$runtimePath -I$executorHeadersPath"
+  val includes = s"-I$runtimePath -I$platformPath -I$executorHeadersPath"
```

Comment:
Include does not recurse by default, we need to edit ```runtimePath``` then if we decide to keep this new file hierarchy in ```runtime``` folder (probably just by concatenating ```/**``` though)

Reply:
I do not think that it is necessary/desirable for it to recurse (i.e. no need for `runtime/ocl/` to be in the include paths). You can still use paths relative to the runtime root as in `#include "gap8/gap8.h"` or  `#include "runtime.h` instead of ` #include "gap8.h"` or `#include "../runtime.h"`.

Reply:
True, I misunderstood this, thinking that you referred to implementation files as well